materialize-firebolt/transactor.go

```diff
@@ -163,16 +163,31 @@ func (t *transactor) projectDocument(spec *pf.MaterializationSpec_Binding, keys
 
 	// Add the keys to the document.
 	for i, value := range keys {
-		var propName = spec.FieldSelection.Keys[i]
+		var propName = strings.ReplaceAll(spec.FieldSelection.Keys[i], "/", "_")
 		document[propName] = value
 	}
 
 	// Add the non-keys to the document.
 	for i, value := range values {
-		var propName = spec.FieldSelection.Values[i]
+		var propName = strings.ReplaceAll(spec.FieldSelection.Values[i], "/", "_")
 
 		if raw, ok := value.([]byte); ok {
-			document[propName] = json.RawMessage(raw)
+			var nestedObject = make(map[string]interface{})
```

Comment:
I think the only values that get encoded as `[]byte` in a `Tuple` are JSON objects and arrays. If that's true, then I think you could just use `document[propName] = string(raw)` and not have to parse anything. The tuple-encoding impl is [here](https://github.com/estuary/flow/blob/master/crates/tuple/src/value.rs#L24), and it appears to be the case that only objects and arrays will end up as byte slices.

Reply:
@psFried Firebolt _does_ support arrays, so for arrays we want to keep them as they are.

Reply:
ah, ok, that makes sense.

Reply:
Ah you are right!